_posts/2019-10-3-Rv.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: Rv
+---
+
+My name is Rv! I am an example here on how your Markdown Profile should look like.
+
+You can use an assortment of Markdown formatting, just ask around and STC will assist you ;)
+
+For example, you can use codes with `\``,` so it'll show up as `hello` rather than hello
+
+Or you can use block quotes with `>`, like this!
+> Hello world!
+
+Finally, here's an image of me!
+![Rv](https://www.google.com/url?sa=i&source=images&cd=&ved=2ahUKEwivqpKPgIDlAhXWiHAKHX3wDAAQjRx6BAgBEAQ&url=https%3A%2F%2Fcherrydeck.com%2F_aravindmoghanphotography_&psig=AOvVaw2fGrcEs1q9pegl40YhFS1i&ust=1570188564179173)
```

Comment:
Your image does load correctly.

Would you like to change the link/image before I approve the merge?

Reply:
I think you meant to say that it doesn't work. 🤔

Reply:
Yea, it doesn't work correctly, sorry haha